### DISCUSSION TITLE
xjbchat/quesbox/

### DISCUSSION THREAD

**Discussion by giscus** [SunnyCloudYang/SunnyCloudYang.github.io#20] [Comments]
# xjbchat/quesbox/

自由选择，随心发问。类GPT，主打一个瞎jb聊。

https://sunnycloudyang.github.io/xjbchat/quesbox/

**Comment by SunnyCloudYang** [comment]
**欢迎提问**！但
一次一个，次数不限，禁止循环，禁止DDoS🙉
Let's begin!

**Comment by ayuusweetfish** [comment]
76, 52, 82, 83选一个（x

**Reply by SunnyCloudYang** [reply]
看来只能83了🤥
喜欢各种奇奇怪怪的有意思的小物件，目前最喜欢的是一块铋晶体和最近买的分光棱镜？感觉很精致，这种晶莹剔透的和光有关的都很喜欢😳

**Reply by ayuusweetfish** [reply]
wow感觉会很好看！|ω･)ﾉ

（然后想问26, 27, 40, 86选一个！

**Reply by SunnyCloudYang** [reply]
86：无聊的时候会干嘛？
想象中：读书，练字，折纸，出去玩，做自己喜欢的事
实际上：打个排位先🤤
